script.js

```diff
@@ -62,19 +63,15 @@
 // =======================================================================
 
 
+// var randomNumber = Math.floor(Math.random()*11)
 
+// var enteredNumber = prompt("Enter a whole number between 1 to 10");
 
```

Comment:
Gotta be careful... randomNumber here goes from 0 to 10, so if it does end up being 0, poor user....